src/index.ts

```diff
@@ -1676,6 +1680,49 @@ class Spanner extends GrpcService {
   static pgJsonb(value): PGJsonb {
     return new codec.PGJsonb(value);
   }
+
+  /**
+   * Helper function to get a Cloud Spanner proMessage object.
+   *
+   * @param {IProtoMessageParams} value The proto message value params in the format of
+   *     @code{IProtoMessageParams}
+   * @returns {ProtoMessage}
+   *
+   * @example
+   * ```
+   * const {Spanner} = require('@google-cloud/spanner');
+   * const protoMessage = Spanner.protoMessage({
+   *   value: singerInfo,
+   *   messageFunction: music.SingerInfo,
+   *   fullName: "spanner.examples.music.SingerInfo"
+   * });
+   * ```
+   */
+  static protoMessage(value: IProtoMessageParams): ProtoMessage {
+    return new codec.ProtoMessage(value);
+  }
+
+  /**
+   * Helper function to get a Cloud Spanner proMessage object.
+   *
+   * @param {IProtoEnumParams} value The proto enum value params in the format of
+   *     @code{IProtoEnumParams}
+   * @returns {ProtoEnum}
+   *
+   * @example
+   * ```
+   * const {Spanner} = require('@google-cloud/spanner');
+   * const protoEnum = Spanner.protoEnum({
+   *   value: '2uhu',
```

Comment:
Normally we expect this value to be a number or number present in string form, right ? 

So try to use a valid value ?

Reply:
We can use number or Enum String.  Updating it to use valid enum string.